server/plugins/client/bundle-handler-factory.js

```diff
@@ -0,0 +1,54 @@
+module.exports.createHandler = createHandler
```

Comment:
Ah - i actually oriented myself in how it was done within the `index.js`:

```
module.exports.register = register
module.exports.register.attributes = {
  name: 'hoodie-local-client',
  dependencies: 'inert'
}
```

But maybe it's different there because it's the index and you add some more attributes to it?
Easy change I can make, though :)